### PR TITLE
Python 3: Replace file() with open()

### PR DESCRIPTION
**Pull request by luckyh**
Signed-off-by: Xu Han <xuhan@redhat.com>

### CODE REVIEW

File: selftests/unit/test_xml_utils.py [hunk 1]
```diff
@@ -63,22 +63,22 @@ def tearDown(self):
     def canonicalize_test_xml(self):
         et = ElementTree.parse(self.XMLFILE)
         et.write(self.XMLFILE, encoding="UTF-8")
-        f = file(self.XMLFILE)
+        f = open(self.XMLFILE)
```
Comment (chunfuwen) [inline selftests/unit/test_xml_utils.py]: Better using with when using open,see https://github.com/avocado-framework/avocado/pull/2319:
'''
-    open(test_log, "a").write('\n' + message + '\n')
+        with open(test_log, "a") as log_file:
+                log_file.write('\n' + message + '\n')
'''

File: selftests/unit/test_xml_utils.py [hunk 2]
```diff
@@ -63,22 +63,22 @@ def tearDown(self):
...
         self.XMLSTR = f.read()
         f.close()
 
     def is_same_contents(self, filename, other=None):
         """Compare filename contents with XMLSTR, or contents of other"""
         try:
-            f = file(filename, "rb")
+            f = open(filename, "rb")
```
Comment (hsmj1412) [inline selftests/unit/test_xml_utils.py]: without close()

File: selftests/unit/test_xml_utils.py [hunk 3]
```diff
@@ -63,22 +63,22 @@ def tearDown(self):
...
             s = f.read()
         except (IOError, OSError):
             logging.warning("File %s does not exist" % filename)
             return False
         if other is None:
             return s == self.XMLSTR
         else:
-            other_f = file(other, "rb")
+            other_f = open(other, "rb")
```
Comment (hsmj1412) [inline selftests/unit/test_xml_utils.py]: without close()

File: shared/scripts/virtio_console_guest.py [hunk 4]
```diff
@@ -1417,9 +1417,9 @@ def daemonize(self):
 
         sys.stdout.flush()
         sys.stderr.flush()
-        si = file(self.stdin, 'r')
-        so = file(self.stdout, 'w')
-        se = file(self.stderr, 'w')
+        si = open(self.stdin, 'r')
+        so = open(self.stdout, 'w')
+        se = open(self.stderr, 'w')
```
Comment (hsmj1412) [inline shared/scripts/virtio_console_guest.py]: without close()

**Pull request merged**